drivers/hmis_csv_importer/app/models/hmis_csv_importer/importer/importer.rb

```diff
@@ -301,7 +510,10 @@ def cleanup_data_set!
     # GrdaWarehouse::Tasks::ServiceHistory::Enrollment.batch_process_date_range!(range)
     # In here, add history_generated_on date to enrollment record
     def ingest!
+      # Reset add/remove counts used for import thresholds
+      reset_import_counts
       importer_log.update(status: :importing)
+      reset_import_counts
```

Comment:
do we actually need to reset this both before and after? It's harmless I guess, just unexpected

Reply:
No!  Thanks for catching that.